databind/src/main/java/tech/ydb/yoj/databind/expression/ModelField.java

```diff
@@ -96,6 +97,10 @@ public FieldValue validateValue(@NonNull FieldValue value) {
             checkArgument(fieldValueType == FieldValueType.BOOLEAN,
                     BooleanFieldExpected::new,
                     p -> format("Specified a boolean value for non-boolean field \"%s\"", p));
+        } else if (value.isByteArray()) {
+            checkArgument(fieldValueType == FieldValueType.BYTE_ARRAY,
+                    ByteArrayFieldExpected::new,
+                    p -> format("Specified a boolean value for non-boolean field \"%s\"", p));
```

Comment:
Typo: "**boolean** value", "non-**boolean** field" instead of "ByteArray value", "non-ByteArray field"